src/app/@ansyn/ansyn/modules/overlays/overlay-status/effects/overlay-status.effects.ts

```diff
@@ -0,0 +1,55 @@
+import { Injectable } from '@angular/core';
+import { CommunicatorEntity, ImageryCommunicatorService, ImageryMapPosition, IMapSettings } from '@ansyn/imagery';
+import {  selectMaps, SetToastMessageAction, UpdateMapAction } from '@ansyn/map-facade';
+import { Actions, Effect, ofType } from '@ngrx/effects';
+import { Dictionary } from '@ngrx/entity';
+import { Store } from '@ngrx/store';
+import { EMPTY, Observable } from 'rxjs';
+import { fromPromise } from 'rxjs/internal-compatibility';
+import { catchError, filter, map, switchMap, withLatestFrom } from 'rxjs/operators';
+import { BackToWorldSuccess, BackToWorldView, OverlayStatusActionsTypes } from '../actions/overlay-status.actions';
+
+
+@Injectable()
+export class OverlayStatusEffects {
+	@Effect()
+	backToWorldView$: Observable<any> = this.actions$
+		.pipe(
+			ofType(OverlayStatusActionsTypes.BACK_TO_WORLD_VIEW),
+			withLatestFrom(this.store$.select(selectMaps)),
+			filter(([action, entities]: [BackToWorldView, Dictionary<IMapSettings>]) => Boolean(entities[action.payload.mapId])),
+			map(([action, entities]: [BackToWorldView, Dictionary<IMapSettings>]) => {
+				const mapId = action.payload.mapId;
+				const selectedMap = entities[mapId];
+				const communicator = this.communicatorsService.provide(mapId);
+				const {position} = selectedMap.data;
+				return [action.payload, selectedMap, communicator, position];
+			}),
+			filter(([payload, selectedMap, communicator, position]: [{ mapId: string }, IMapSettings, CommunicatorEntity, ImageryMapPosition]) => Boolean(communicator)),
+			switchMap(([payload, selectedMap, communicator, position]: [{ mapId: string }, IMapSettings, CommunicatorEntity, ImageryMapPosition]) => {
+				const disabledMap = communicator.activeMapName === 'disabledOpenLayersMap';
```

Comment:
use DisabledOpenLayersMapName instaed of hard coded string 'disabledOpenLayersMap'